src/server/schema.js

```diff
@@ -42,60 +47,154 @@ const gqlTypeToAggsHistogramName = {
 
 const getAggsHistogramName = (gqlType) => {
   if (!gqlTypeToAggsHistogramName[gqlType]) {
-    throw new Error(`Invalid elasticsearch type ${gqlType}`);
+    // throw new Error(`Invalid elasticsearch type ${gqlType}`);
+    return '';
   }
   return gqlTypeToAggsHistogramName[gqlType];
 };
 
-const getQuerySchemaForType = (esType) => {
+const getQuerySchemaForType = (esInstance, esIndex, esType) => {
   const esTypeObjName = firstLetterUpperCase(esType);
-  return `${esType} (
+  const fieldESTypeMap = esInstance.getESFieldTypeMappingByIndex(esIndex);
+  const existingFields = new Set([]);
+
+  const queueFields = [];
+  Object.keys(fieldESTypeMap).forEach((fieldKey) => {
+    const esFieldType = fieldESTypeMap[fieldKey].type;
+    if (esFieldType === 'nested' && !existingFields.has(fieldKey)) {
+      queueFields.push({ key: fieldKey, value: fieldESTypeMap[fieldKey] });
+      existingFields.add(fieldKey);
+    }
+  });
+  let sType = `${esType} (
     offset: Int, 
     first: Int,
     filter: JSON,
     sort: JSON,
     accessibility: Accessibility=all,
     ): [${esTypeObjName}]`;
+  while (queueFields.length > 0) {
+    const f = queueFields.shift();
+    sType += `
```

Comment:
Adding the nested fields in top-level query schema is really working: so if I query nested fields in top level like: ```{
  ActionableMutations {
    BxMorphology
  }
}```  it will return null:  ```{
  "data": {
    "ActionableMutations": null
  }
}```

Reply:
that type is necessary for declaring nested field